src/core/playground/accountAuthenticator/legacyMultiEd25519.ts

```diff
@@ -0,0 +1,54 @@
+import type { Deserializer, Serializer } from "../../../bcs";
+import { HexInput } from "../../../types";
+import type { Ed25519Signature } from "../ed25519";
+import { LegacyMultiEd25519PublicKey, LegacyMultiEd25519Signature } from "../legacy/multiEd25519";
+import { LegacyAccountAuthenticatorEd25519 } from "./legacyEd25519";
+import { AccountAuthenticatorVariant } from "./variant";
+
+export class LegacyAccountAuthenticatorMultiEd25519 {
+  public readonly publicKey: LegacyMultiEd25519PublicKey;
+
+  public readonly signature: LegacyMultiEd25519Signature;
+
+  constructor(publicKey: LegacyMultiEd25519PublicKey, signature: LegacyMultiEd25519Signature);
+  constructor(publicKey: LegacyMultiEd25519PublicKey, authenticators: LegacyAccountAuthenticatorEd25519[]);
+  constructor(
+    publicKey: LegacyMultiEd25519PublicKey,
+    signatureOrAuthenticators: LegacyMultiEd25519Signature | LegacyAccountAuthenticatorEd25519[],
```

Comment:
why the function overload? and the 2 return types?

Reply:
One cool aspect of it is that the user doesn't have to worry about creating the bitmap :)

Reply:
> and the 2 return types

which return types?

Reply:
sorry, input types 
`signatureOrAuthenticators: LegacyMultiEd25519Signature | LegacyAccountAuthenticatorEd25519[]`

Reply:
This overload is kinda weird with `signaureOrAuthenticators`

Why not just:

(publicKey, signature)
(publicKey, authenticators)

And then do whatever type magic to handle one or the other?

Reply:
That's the actual implementation, that needs to take into account all overloads unfortunately (it's still JS at the end of the day)

The only possible overloads are 
```typescript
  constructor(publicKey: LegacyMultiEd25519PublicKey, signature: LegacyMultiEd25519Signature);
  constructor(publicKey: LegacyMultiEd25519PublicKey, authenticators: LegacyAccountAuthenticatorEd25519[]);
```

when definining overloads in TS, the main implementation is actually not available as overload, and would need to be specified explicitly.